components/scripts/lib/exit-code.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+SUCCESS=0
+INVALID_INPUT=1
+UNEXPECTED_ERROR=2
```

Comment:
I tried swapping 1 and 2, but unfortunately, the code argbash generates always uses exit code 1 for unknown arguments (or for any argument validation).  I'll keep digging a bit to see if that can be influenced or changed.

Reply:
Got it. I would not break a leg over this.